package.json

```diff
@@ -6,17 +6,15 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "format:code": "prettier --write .",
+    "format": "prettier --write .",
     "lint": "next lint",
-    "test": "jest --watch",
+    "postinstall": "husky install",
+    "test": "jest --coverage",
+    "test:watch": "jest --watch",
     "test:ci": "jest --ci",
-    "e2e": "cypress run",
-    "cy:open": "cypress open"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
```

Comment:
This is no longer needed, see [here](https://blog.typicode.com/husky-git-hooks-javascript-config/).